### PR TITLE
Movimiento básico A-D

### PR DESCRIPTION
**Pull request by JesusRuiz02**
Se agregaron los movimientos básicos de izquierda y derecha

### CODE REVIEW

File: Assets/Scripts/PlayerMovement.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] 
+    private int _speed = 3; 
+    Rigidbody2D rb2d;
+    // Start is called before the first frame update
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs:11]: Remove comment

File: Assets/Scripts/PlayerMovement.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    void Start()
+    {
+        rb2d = GetComponent<Rigidbody2D>();
+    }
+
+    // Update is called once per frame
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs:17]: Remove comment

File: Assets/Scripts/PlayerMovement.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField]
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs:8]: Move `SerializeField` to same line as `_speed`

File: Assets/Scripts/PlayerMovement.cs [hunk 4]
```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] 
+    private int _speed = 3; 
+    Rigidbody2D rb2d;
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs:10]: Add access modifier and default value, also add. aline between start and field.

File: Assets/Scripts/PlayerMovement.cs [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    // Start is called before the first frame update
+    void Start()
+    {
+        rb2d = GetComponent<Rigidbody2D>();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs:20]: Remove whitespace

File: Assets/Scripts/PlayerMovement.cs [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+        rb2d = GetComponent<Rigidbody2D>();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+       
+        if (Input.GetKey(KeyCode.A))
+        {
+            transform.position = transform.position + new Vector3(-(_speed)*Time.deltaTime,0,0);
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs:23]: This line is not formatted, the values inside `Vector3.x` are too clunky, see Microsoft docs or auto format tool on your IDE.

File: Assets/Scripts/PlayerMovement.cs [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+        }
+
+        if (Input.GetKey(KeyCode.D))
+        {
+           transform.position = transform.position + new Vector3(_speed*Time.deltaTime,0,0);
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs:28]: Same as above and turn these two movement lines into a single function with parameter for direction. Can be `bool` or `int`.

File: Assets/Scripts/PlayerMovement.cs [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+        }
+
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs:30]: Remove whitespace.

File: Assets/Scripts/PlayerMovement.cs [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    void ControllerPlayer()
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs:33]: Remove empty method.

File: Assets/Scripts/PlayerMovement.cs [hunk 10]
```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs:3]: Remove unused libraries. You can modify default script in Unity system files. (Ask teacher on Discord)

**Review by Horothenic — CHANGES_REQUESTED** [review]

File: Assets/Scripts/PlayerMovement.cs (hunk 6 above)
Comment (JesusRuiz02) [reply Assets/Scripts/PlayerMovement.cs:23]: What do you mean about clunky?, it's too basic to use the values of 0,0?

Comment (Horothenic) [reply Assets/Scripts/PlayerMovement.cs]: This code `new Vector3(-(_speed)*Time.deltaTime,0,0)` does not follow the C# guidelines for operator separation, this should be like this `new Vector3(-_speed * Time.deltaTime, 0, 0)`, there are some rules that all the code in C# Microsoft docs respect, like space after commas on parameters or space on each side on math or boolean operators.

File: Assets/Scripts/PlayerMovement.cs [hunk 11]
```diff
@@ -1,37 +1,36 @@
 using System;
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
-
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs]: This whitespace is ok, to separate libraries from the class.

File: Assets/Scripts/PlayerMovement.cs [hunk 12]
```diff
@@ -1,37 +1,36 @@
...
 public class PlayerMovement : MonoBehaviour
-{
-    [SerializeField] 
-    private int _speed = 3; 
-    Rigidbody2D rb2d;
-    // Start is called before the first frame update
+{ 
+    [SerializeField] private int _speed = 3;
+    private bool Direction = true;
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs:6]: The convention for names should be `_direction`

File: Assets/Scripts/PlayerMovement.cs [hunk 13]
```diff
@@ -1,37 +1,36 @@
...
+    private Rigidbody2D Rb2d = null;
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs:7]: The convention for names should be `_rigidbody2D`, also add a space between fields and methods.

File: Assets/Scripts/PlayerMovement.cs [hunk 14]
```diff
@@ -1,37 +1,36 @@
...
     void Start()
     {
-        rb2d = GetComponent<Rigidbody2D>();
+        Rb2d = GetComponent<Rigidbody2D>();
     }
-
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs]: Leave space between methods.

File: Assets/Scripts/PlayerMovement.cs [hunk 15]
```diff
@@ -1,37 +1,36 @@
...
-        if (Input.GetKey(KeyCode.A))
+      if (Input.GetKey(KeyCode.A))
+      {
+        Direction = true;
+        PlayerController(Direction, _speed);
+      }
+      if (Input.GetKey(KeyCode.D))
+      {
+        Direction = false;
+        PlayerController(Direction, _speed);
```
Comment (Horothenic) [inline Assets/Scripts/PlayerMovement.cs:22]: You don't need `Direction`, you can write `true` or `false` directly on method.

File: Assets/Scripts/PlayerMovement.cs (hunk 12 above)
Comment (Horothenic) [reply Assets/Scripts/PlayerMovement.cs:6]: And I this is not needed.

**Review by Horothenic — CHANGES_REQUESTED** [review]

File: Assets/Scripts/PlayerMovement.cs (hunk 6 above)
Comment (JesusRuiz02) [reply Assets/Scripts/PlayerMovement.cs]: ohhh okay, thank you

**Review by Horothenic — APPROVED** [review]

**Pull request merged**